.github/workflows/reusable-build-test-release.yml

```diff
@@ -1568,7 +1586,7 @@ jobs:
         with:
           splunk: ${{ matrix.splunk.version }}${{ secrets.OTHER_TA_REQUIRED_CONFIGS }}
           test-type: ${{ env.TEST_TYPE }}
-          test-args: ${{ env.TEST_ARGS }}
+          test-args: ${{ env.TEST_ARGS }} ${{ steps.create-test-arg.outputs.test-arg }} ${{ matrix.marker }}
           job-name: ${{ steps.create-job-name.outputs.job-name }}
```

Comment:
Isn't test-browser input missing here?

Reply:
@kdoroszko-splunk Or rather you want to onboard this change with another PR?

Reply:
Let's merge this PR. I've added the argument here: [line1](https://github.com/splunk/addonfactory-workflow-addon-release/pull/261/files#diff-bdb402178d2b76aa0f8acc15af0ee19e0bd76fd53028475dabe5503b5fe874dcR1514) [line2](https://github.com/splunk/addonfactory-workflow-addon-release/pull/261/files#diff-bdb402178d2b76aa0f8acc15af0ee19e0bd76fd53028475dabe5503b5fe874dcR1601). 
I will also have to change splunk/wfe-test-runner-action@v1.6 version to v5.0 after merging this [PR](https://github.com/splunk/wfe-test-runner-action/pull/27).

Reply:
@kdoroszko-splunk please update documentation as well accordingly.

Reply:
@mkolasinski-splunk, done. Take a look.